src/gui/guiFormSpecMenu.cpp

```diff
@@ -1942,6 +1942,9 @@ void GUIFormSpecMenu::parseTabHeader(parserData* data, const std::string &elemen
 			pos = getRealCoordinateBasePos(v_pos);
 
 			geom = getRealCoordinateGeometry(v_geom);
+			// Set default height
+			if (parts.size() == 6)
```

Comment:
This should be `parts.size() <= 6`, because this should apply to the 4-part formspec too